Makefile.PL

```diff
@@ -125,9 +133,9 @@ WriteMakefile(
       resources =>
       {
         license     => 'http://dev.perl.org/licenses/',
-        bugtracker  => 'https://github.com/rurban/Cpanel-JSON-XS/issues',
+        bugtracker  => { web => 'https://github.com/rurban/Cpanel-JSON-XS/issues' },
         # Note: https://rt.cpan.org/Public/Dist/Display.html?Queue=Cpanel-JSON-XS is also observed
-        repository  => 'https://github.com/rurban/Cpanel-JSON-XS',
+        repository  => { url => 'https://github.com/rurban/Cpanel-JSON-XS' },
```

Comment:
This should have at the least `type => 'git'`.

Reply:
fixed